schemes/musica/configurations/chapman/micm/config.json

```diff
@@ -3,4 +3,4 @@
     "species.json",
     "reactions.json"
   ]
-}
+}
```

Comment:
Can I ask if there's a particular reason trailing empty lines are removed from here (and missing in other files)?

Reply:
no reason, just didn't notice it. I'll add it back in.